### PR TITLE
bugfix external link menu management item

### PR DESCRIPTION
**Pull request by miftahurrahmi**

### CODE REVIEW

File: src/resources/js/components/BadasoSidebarItem.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     :class="{ 'vs-sidebar-item-active': getActive }"
-    class="vs-sidebar--item"
+    class="vs-sidebar--item "
```
Comment (rizkiheryandi) [inline src/resources/js/components/BadasoSidebarItem.vue:4]: unwanted space ?

File: src/resources/js/components/BadasoSidebarMenu.vue [hunk 2]
```diff
@@ -25,6 +25,15 @@
         />
       </a>
       <!-- for route link -->
+      <template v-else>
+        <a v-if="url.substring(0, 4) == 'http'"
```
Comment (rizkiheryandi) [inline src/resources/js/components/BadasoSidebarMenu.vue:29]: how about https ?

File: src/resources/js/components/BadasoSidebarItem.vue (hunk 1 above)
Comment (miftahurrahmi) [reply src/resources/js/components/BadasoSidebarItem.vue:4]: sorry, it was a typo and I've returned to the original position

File: src/resources/js/components/BadasoSidebarMenu.vue (hunk 2 above)
Comment (miftahurrahmi) [reply src/resources/js/components/BadasoSidebarMenu.vue:29]: without any https conditions the condition function produces the same results for http or https

**Review by rizkiheryandi — APPROVED** [review]

**Pull request merged**